using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,117 @@
+# Using the Portaal taak Plugin
+
+The portaal taak plugin is used to create taak objects in the Objecten API for a BPMN user task so they can be 
+completed through other applications like the NL Portal. This allows Valtimo to delegate tasks to other systems
+that are better suited to execute these tasks, or allow access from users that do not have access to the Valtimo
+frontend.
+
+## How does the plugin work
+
+The lifecycle of a portaal taak is as follows:
+1. When a process instance reaches a user task configured with the [Create Portaal taak](#create-portaal-taak) action
+an instance of the task is created in valtimo as usual. Additionally a taak object is created in the Objecten API of
+object type `Taak`. The status of the taak object will be `open`. The taak object includes information on who the 
+intended recipient is. This is the person or organization that is supposed to complete the task.
+2. External applications, such as the NL Portal, can retrieve the task information and offer a way to 
+complete the task. Often this will be a form to input data. 
+3. When the user completes the task in the external system, this system will update the taak object with the new data
+and change the status in the object to `ingediend`.
+4. The Objecten API should send a notification to the Notificaties API indicating the the object has been updated.
+5. Valtimo should be subscribed to notifications from the Notificaties API as part of the plugin setup and will receive
+the notification. When an update notification for a taak object is received Valtimo will check the status. If this is 
+set to `ingediend` the updated data from the taak object will be stored, and the handling process will be started.
+6. As part of the handling process the [Complete Portaal taak](#complete-portaal-taak) action should be used. This 
+action will complete the task in Valtimo and update the zaak object status to `verwerkt`, indicating the the taak has 
+been processed. 
+   
+## Configure the plugin
+
+A plugin configuration is required before the plugin can be used. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Portaal taak plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the Zaken API dependency can be found
+[here](/getting-started/modules/zgw/portaaltaak.md).
+
+To configure this plugin the following properties have to be entered:
+- **Notification API plugin.** Reference to another plugin configuration that will be used to receive notification on
```

Comment:
Shouldn't 'Notification API' be in Dutch?

Reply:
Arguably. I used the variable names as they are displayed on the screen when using the english language settings.